### PR TITLE
docs : new known rpcerrors

### PR DESCRIPTION
**Pull request by New-dev0**

### CODE REVIEW

**Comment by Lonami** [comment]
Can you also add `MEDIA_GROUPED_INVALID`, caused by `editMessage`? ([source](https://t.me/TelethonChat/338001))

**Comment by New-dev0** [comment]
Added 👍

File: telethon_generator/data/methods.csv [hunk 1]
```diff
@@ -192,7 +192,7 @@ messages.editChatDefaultBannedRights,both,BANNED_RIGHTS_INVALID
 messages.editChatPhoto,both,CHAT_ID_INVALID INPUT_CONSTRUCTOR_INVALID INPUT_FETCH_FAIL PEER_ID_INVALID PHOTO_EXT_INVALID
 messages.editChatTitle,both,CHAT_ID_INVALID NEED_CHAT_INVALID
 messages.editInlineBotMessage,both,MESSAGE_ID_INVALID MESSAGE_NOT_MODIFIED
-messages.editMessage,both,CHANNEL_INVALID CHANNEL_PRIVATE CHAT_WRITE_FORBIDDEN INLINE_BOT_REQUIRED INPUT_USER_DEACTIVATED MEDIA_NEW_INVALID MEDIA_PREV_INVALID MESSAGE_AUTHOR_REQUIRED MESSAGE_EDIT_TIME_EXPIRED MESSAGE_EMPTY MESSAGE_ID_INVALID MESSAGE_NOT_MODIFIED PEER_ID_INVALID
+messages.editMessage,both,CHANNEL_INVALID CHANNEL_PRIVATE CHAT_WRITE_FORBIDDEN INLINE_BOT_REQUIRED INPUT_USER_DEACTIVATED MEDIA_NEW_INVALID MEDIA_PREV_INVALID MESSAGE_AUTHOR_REQUIRED MESSAGE_EDIT_TIME_EXPIRED MESSAGE_EMPTY MESSAGE_ID_INVALID MESSAGE_NOT_MODIFIED PEER_ID_INVALID MEDIA_GROUPED_INVALID
```
Comment (Lonami) [inline telethon_generator/data/methods.csv:195]: I'm trying to keep this list sorted:

```suggestion
messages.editMessage,both,CHANNEL_INVALID CHANNEL_PRIVATE CHAT_WRITE_FORBIDDEN INLINE_BOT_REQUIRED INPUT_USER_DEACTIVATED MEDIA_GROUPED_INVALID MEDIA_NEW_INVALID MEDIA_PREV_INVALID MESSAGE_AUTHOR_REQUIRED MESSAGE_EDIT_TIME_EXPIRED MESSAGE_EMPTY MESSAGE_ID_INVALID MESSAGE_NOT_MODIFIED PEER_ID_INVALID
```

File: telethon_generator/data/errors.csv [hunk 2]
```diff
@@ -175,6 +175,7 @@ MAX_QTS_INVALID,400,The provided QTS were invalid
 MD5_CHECKSUM_INVALID,400,The MD5 check-sums do not match
 MEDIA_CAPTION_TOO_LONG,400,The caption is too long
 MEDIA_EMPTY,400,The provided media object is invalid or the current account may not be able to send it (such as games as users)
+MEDIA_GROUPED_INVALID,400,The Grouped Media is Invalid
```
Comment (Lonami) [inline telethon_generator/data/errors.csv:178]: I'd rather not provide any description if it's just going to repeat the error message (I know there are other older cases doing this, but I want to move away from that):

```suggestion
MEDIA_GROUPED_INVALID,400,
```

**Review by Lonami — CHANGES_REQUESTED** [review]

**Comment by Lonami** [comment]
Thanks.

**Pull request merged**